backend/src/main/java/com/huellapositiva/infrastructure/SecurityConfig.java

```diff
@@ -73,7 +73,7 @@ protected void configure(HttpSecurity http) throws Exception {
         http.cors().and()
                 .csrf()
                 .ignoringAntMatchers("/api/v1/authentication/login", "/api/v1/volunteers", "/api/v1/contactperson")
```

Comment:
Why do we ignore this paths? A CSRF token should be mandatory in any state changing request unless we have a good reason not to.